hera_qm/firstcal_metrics.py

```diff
@@ -298,17 +266,20 @@ def plot_zscores(metrics, fname=None, plot_type='full', ax=None, figsize=(10, 6)
 
 
 class FirstCal_Metrics(object):
-    """
+    """Object to store and compute FirstCal metric data.
+
     FirstCal_Metrics class for holding firstcal data,
     running metrics, and plotting delay solutions.
     Currently only supports single polarization solutions.
     """
 
     # sklearn import statement
     sklearn_import = sklearn_import
+    jones2pol = {-5: 'XX', -6: 'YY', -7: 'XY', -8: 'YX'}
```

Comment:
Should we import these from pyuvdata instead? Just to make sure that we're being totally self-consistent?

Reply:
That's a good idea.